app/screens/index.tsx

```diff
@@ -262,6 +262,12 @@ Navigation.setLazyComponentRegistrator((screenName) => {
         case Screens.CALL:
             screen = withServerDatabase(require('@calls/screens/call_screen').default);
             break;
+        case Screens.CONVERT_GM_TO_CHANNEL:
+            screen = withServerDatabase(require('@screens/convert_gm_to_channel').default);
+            break;
+        case Screens.TEAM_SELECTOR_LIST:
+            screen = withServerDatabase(require('@screens/convert_gm_to_channel/team_selector_list').default);
+            break;
```

Comment:
I know call is out of place, but let's try to keep this file alphabetically sorted.

Reply:
Done, although I see no benefit in always doing this manually. I'm 4/5 that if an arbitrary code convention isn't enforced and preferably auto-fixed by linters, it shouldn't be cared about.